.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: true,
+    trailingComma: "all",
+    singleQuote: true,
+    printWidth: 120,
+    tabWidth: 4
```

Comment:
I would recommend to use `tabWith: 2`. 

What kind of indent is used here `tab` or `space`? If `tab` that it should be changed to `space`.